easylistdutch/block_third_party_resource.txt

```diff
@@ -1,122 +1,36 @@
 .nl/adurl/$xmlhttprequest,domain=start-player.npo.nl
-||109.70.1.61/bannersystem/
-||actiecode.nl/dynamics/modules/spub$third-party
 ||adslwinkel.nl/templates/adslwinkel/images/adslnl.gif$domain=htforum.nl
 ||amz01.plzcdn.com/a/
-||antares.fm/banners/
-||api.hebbes.be/widgets/carousels/$third-party
 ||api.skyscanner.net/api.ashx?$domain=viva.nl
-||babysitster.nl/banners/
 ||blablacar.nl/widget-driver/$third-party
 ||bol.com/nl/upload/banners/
 ||booking.com/?aid=$domain=puuropreis.nl
 ||brightcove.com/js/BrightcoveExperiences.js$domain=backpackgek.nl
-||buzzlab.nl/input/*Banner
+||cd.tnet.nl/ads/ads.js$script,important,domain=www.metronieuws.nl|www.telegraaf.nl
 ||cdn.media.zylom.com/images/site/zylom/promo/$third-party
 ||centraal.helpmij.nl/images/nieuwsbrief/nod32-colofon.jpg
-||cityfashion.be/external/portal/
-||cupaa.nl/banners/
-||de-keizer.nl/tassen/*/banner
-||demediahub.nl^*/media/commercial/
-||doubleclick.net/pfadx/homepage.nos.nl/
-||doubleclick.net/pfadx/mediaplayer.nl/
-||doubleclick.net/pfadx/npoplayer.nl/
-||doubleclick.net/pfadx/vara.homepage.nl/
-||doubleclick.net/pfadx/vnumedia.tweak
+||config-prod.choice.faktor.io/*/faktor.js$script,domain=consent.talpanetwork.com
```

Comment:
Ì don't see this one anymore

Reply:
Because some cookiebanners uses talpanetwork.com, and talpanetwork.com uses "config-prod.choice.faktor.io..." this should be blocked.